mypy.ini

```diff
@@ -0,0 +1,37 @@
+
+[mypy]
+python_version = 3.7
+exclude = (?x)(
+    tests/    # test directory
+  )
+pretty = True
+
+# Per-module options:
```

Comment:
duplicate comment on line 10?

Reply:
Yup. Good catch. I'll remove this in another PR that's pending on Stash.